.vib/kong/cypress/cypress/integration/kong_spec.js

```diff
@@ -51,6 +51,14 @@ it('allows adding a new service and route through admin API', () => {
       expect(response.status).to.eq(201);
     });
 
+    cy.request({
+      method: 'GET',
+      url: `${BASE_URL}/services`,
+    }).then((response) => {
+      expect(response.status).to.eq(200);
+      expect(JSON.stringify(response.body)).to.contain(services.newService.upstreamURL.replace(/^https?:\/\//, ''));
```

Comment:
Not really an issue but just a FYI, I'll say there is no need to use `JSON.stringify()` for the response's body.

Reply:
The response is an object and we need to make it a string for this to work.